about.html

```diff
@@ -1,103 +1,147 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+        integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" rel="stylesheet" />
+    <link rel="stylesheet" href="about.css" />
+    <title>About</title>
 </head>
+
 <body>
-  <header>
-    <div class="topbar">
-        <a href=""><i class="fa-brands fa-fecebook-f"></i></a>
-        <a href=""><i class="fa-brands fa-twitter"></i></a>
-        <a href="">Sign up</a>
-        <a href="">Contact</a>
-    </div>
-    <nav class="navmenu-desktop">
-        <a id="sitelogo" href="./index.html"><img src="./img/Music logo.jpg" alt="Music logo"></a>
-        <ul class="desktopnav">
-            <a href="./index.html">
-                <li>Home</li>
-            </a>
-            <a href="./about.html">
-               <li>About</li>
-            </a>
-            <a href="./contact.html">
-               <li>Contact</li>
-            </a>
-            <a href=""><button type="button">Program</button></a>
-        </ul>
-    </nav>
-    <i class="fa-solid fa-bars" onclick="openMenu()" id="menu-icon"></i>
-  </header> 
+    <header>
+        <div class="newtopbar">
+            <a href=""><i class="fa-brands fa-facebook-f"></i></a>
+            <a href=""><i class="fa-brands fa-twitter"></i></a>
+            <a href="">Logout</a>
+            <a href="">Contact</a>
+            <a href="">Morocco</a>
+        </div>
+        <nav class="navmenu-desktop">
+            <a id="sitelogo" href="./index.html"><img src="./img/h-music-logo.jpg" id="logoimage"
+                    alt="Music logo" /></a>
+            <ul class="desktopnav">
+                <a href="./index.html">
+                    <li>Home</li>
+                </a>
+                <a id="activelink" href="./about.html">
+                    <li>About</li>
+                </a>
+                <a href="">
+                    <li>Sponsor</li>
+                </a>
+                <a href="">
+                    <li>News</li>
+                </a>
+                <a href=""><button type="button">Campaign</button></a>
+            </ul>
+        </nav>
+        <i class="fa-solid fa-bars" onclick="OpenMenu()" id="menu-icon"></i>
+    </header>
 
-  <div class="menuformobile">
-    <img class="X-icon" onclick="CloseMenu()" src="./img/X-icon.svg" alt="X icon">
-    <div class="top-menu">
-        <ul>
-            <li><a><i class="fa-brands fa-facebook"></i></a></li>
-            <li><a><i class="fa--brands fa-twitter"></i></a></li>
-            <li><a>Sign in</a></li>
-            <li><a>contact</a></li>
+    <!-- mobile menu -->
+    <div class="menuformobile">
```

Comment:
- [ ] On the Mobile version, the background of About should be black as per the requirements.